### DISCUSSION TITLE
LaTeX-incompatible input and strict mode is set to 'warn': Too few columns specified in the {array} column argument. [textEnv]

### DISCUSSION THREAD

**Discussion by sebastianjhm** [KaTeX/KaTeX#3678] [Q&A]
I want to render this expression using katex.renderToString() in node
```tex
F_{X}\left(x\right)=\left\{\begin{array}{cl} \exp\Bigl(-\exp(-z(x))\Bigr) & \text{if } ~ \xi=0 \\ \exp\Bigl(-(1+\xi z(x))^{-1/\xi}\Bigr) & \text{if } ~ \xi \neq 0\end{array} \right.\\
```
In notion I see:
![image](https://user-images.githubusercontent.com/53499619/183266746-1ab8827e-0c23-466a-80ab-0149d22333f0.png)
But in my project:
![image](https://user-images.githubusercontent.com/53499619/183266767-2c37fb8c-99cb-4851-a261-3f0436b63271.png)
In console appear:
![image](https://user-images.githubusercontent.com/53499619/183266779-2f99a071-fa06-40d7-ae2d-608e252752a5.png)

**Comment by edemaine** [comment]
It seems to render fine on [the demo site](https://katex.org):

![image](https://user-images.githubusercontent.com/2218736/183271087-73b860e5-4ad6-4c21-810b-185debce9caa.png)

So this is probably something specific to your project. My guess would be that you're preprocessing the content in some way (e.g. Markdown) that converts `\\` into `\`. You could try `\\\\` as a workaround, or turn off the preprocessing (at least within math expression), or better integrate KaTeX into that preprocessing so that it doesn't apply to math expressions.

**Reply by vladtimss** [reply]
@SebastianJHM did you solve this problem? could you give me advise?
